app/soundcloud.js

```diff
@@ -75,9 +75,11 @@ module.exports = class SoundCloud extends Events {
       titleParts = title.split(' in ', 2)
     if (titleParts.length == 2)
       // Title has " in " in it when not playing but on a playlis page
-      this.isPlaying().then(isPlaying => {
+      this.isPlaying().then(obj => {
```

Comment:
@salomvary I was able to make all the changes you suggested except here. When I tried
```javascript
this.isPlaying().then({isPlaying, icon} => {
```
I received an error.